tests/test_core/test_alignment.py

```diff
@@ -1510,6 +1510,15 @@ def test_construction(self):
         seq2 = make_seq("AC", name="seq2")
         coll = SequenceCollection(data=[seq1, seq2])
 
+    def test_write_to_json(self):
+        # test writing to json file
+        aln = self.Class([("a", "AAAA"), ("b", "TTTT"), ("c", "CCCC")])
+        fn = mktemp(suffix=".json")
```

Comment:
I think it's better to use a temporary directory as a context, since the directory and all it's contents cleaned up after exiting the context, e.g.

```python
with TemporaryDirectory('.') as dirname:
    fn = pathlib.Path(dirname) / "sample.json" # use pathlib because it handles windows/linux paths
    # other operations

# and here the directory and all it's contents have been deleted
```